index.html

```diff
@@ -34,5 +34,296 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
+                            <li><a href="#" target="_blank"><img src="./assets/images/medium.svg" alt=""></a></li>
+                            <li><a href="https://github.com/jlcarrascof" rel="noopener" target="_blank"><img src="./assets/images/github.svg" alt=""></a></li>
+                            <li><a href="#" target="_blank"><img src="./assets/images/angel-list.svg" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div>
+                    <button class="btn">Get my resume</button>
+                </div>
+            </div>
+            <div class="skill-container">
+                <ul>
+                    <li class="language-container">
+                        <div class="skill-item">
+                            <p>Languages</p>
+                            <button>
+                                <img src="/assets/images/arrow-down.png" alt="arrow icon">
+                            </button>
+                        </div>
+                        <div>
+                            <ul class="language-list">
+                                <li class="language-item">
+                                    <div>
+                                        <img src="/assets/images/logo-js.png" alt="js icon">
+                                        <p>Javascript</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="/assets/images/logo-html.png" alt="html icon">
+                                        <p>HTML</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="/assets/images/logo-css.png" alt="css icon">
+                                        <p>CSS</p>
+                                    </div>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    
+                    <li class="skill-item bottom-bar">
+                        <p>Frameworks</p>
+                        <button>
+                            <img src="/assets/images/arrow-right.png" alt="arrow icon">
+                        </button>
+                    </li>
+                    <li class="skill-item bottom-bar">
+                        <p>Skills</p>
+                    <button>
+                        <img src="/assets/images/arrow-right.png" alt="arrow icon">
+                    </button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please try to match your design to the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_myself](https://user-images.githubusercontent.com/70483835/227731334-56a3ef9f-a5b8-4eca-b61b-3663f2441ffa.png) | ![figma_myself](https://user-images.githubusercontent.com/70483835/227731330-ef3d6575-2a4a-4cdb-9f59-26136ef5160c.png)

Reply:
And finally and hopefully, this one too.